website/project/metadata/egap-registration.json

```diff
@@ -0,0 +1,418 @@
+					"qid": "q24",
+					"title": "C7 Has this research received Institutional Review Board (IRB) or ethics committee approval?",
+					"nav": "Review Board (IRB) or ethics committee approval?",
+					"type": "choose",
+					"format": "singleselect",
+					"options": [
+						"N/A",
+						"No",
+						"Yes",
+						"Other"
+					]
+				},
+				{
+					"qid": "q25",
+					"title": "Other",
+					"format": "textarea",
+					"required": false
+				},
+				{
+					"qid": "q26",
+					"title": "C8 IRB Number",
+					"nav": "IRB Number",
+					"type": "string",
+					"format": "text"
+				},
+				{
+					"qid": "q27",
+					"title": "C9 Date of IRB Approval",
+					"nav": "IRB Number",
+					"type": "string",
+					"format": "text"
+				},
+				{
+					"qid": "q28",
+					"title": "C10 Will the intervention be implemented by the researcher or a third party? If a third party, please provide the name.",
+					"nav": "Review Board (IRB) or ethics committee approval?",
+					"type": "choose",
+					"format": "singleselect",
+					"options": [
+						"Researchers",
+						"Other"
+					]
+				},
+				{
+					"qid": "q29",
+					"title": "Other",
+					"format": "textarea",
+					"required": false
+				},
+				{
+					"qid": "q30",
+					"title": "C11 Did any of the research team receive remuneration from the implementing agency for taking part in this research?",
+					"nav": "Remuneration?",
+					"type": "choose",
+					"format": "singleselect",
+					"options": [
+						"N/A",
+						"Yes",
+						"No",
+						"Other"
+					]
+				},
+				{
+					"qid": "q31",
+					"title": "Other",
+					"format": "textarea",
+					"required": false
+				},
+				{
+					"qid": "q32",
+					"title": "C12 If relevant, is there an advance agreement with the implementation group that all results can be published?",
+					"nav": "is there an advance agreement with the implementation group that all results can be published?",
+					"type": "choose",
+					"format": "singleselect",
+					"options": [
+						"N/A",
+						"Yes",
+						"No",
+						"Other"
+					]
+				},
+				{
+					"qid": "q33",
+					"title": "Other",
+					"format": "textarea",
+					"required": false
+				},
+				{
+					"qid": "q34",
+					"title": "C13 JEL classification(s)",
+					"nav": "JEL classification(s)",
+					"type": "string",
+					"format": "text",
+					"description": "Please list any relevant classifications from the Journal of Economic Literature. If multiple classifications, separate by commas (e.g. D31, C19, F22)"
+				}
+			]
+		},
+		{
+			"id": "page3",
+			"title": "Keywords and Data",
+			"questions": [{
+				"qid": "q35",
+				"nav": "Keywords",
+				"type": "choose",
+				"format": "multiselect",
+				"title": "Keywords for Methodology",
+				"description": "Choose one or more categories that describe your study methodology.",
+				"options": [
+					"Experimental Design",
+					"Lab Experiments",
+					"Experimental Design",
+					"Statistics",
+					"Survey Methodology"
```

Comment:
This is missing some options from google doc - Field Experiments. Mixed Method.  Experimental design shows up twice.